### ISSUE TITLE
Confidence/error bounds for physical scale

### ISSUE THREAD

**Issue by azaroth42** [IIIF/iiif.io#293]
Optional field in the physical dimensions service to allow the recording of how confident the service is in the scale.

For example, if it's from the PPI of an image that's set as 300 for every single image from the repository, it should probably be 0.  If it's automatically set from some service that generates it by image analysis on a ruler or other target object, maybe it gets somewhere between 0.4 and 0.7, and if it has been manually checked, maybe it's 1.0 ?

Basically ... how much should I believe you when you say this, knowing that there's a lot of untrustworthy data.

So my proposal is "certainty", range of 0.0 to 1.0.

**Comment by azaroth42** [comment]
(non-breaking so resetting milestone to 2.2 so we reassess then)

**Comment by azaroth42** [comment]
Propose close, no one cares enough about the service let alone the confidence.

**Comment by zimeon** [comment]
:+1: wontfix

**Comment by jpstroop** [comment]
:+1:

**Comment by mikeapp** [comment]
👍  won't fix

**Comment by zimeon** [comment]
Agreed at Hague, close wontfix
